app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -218,5 +219,11 @@ def parse_facilities(facilities)
         )
       end
     end
+
+    def dedupe_facilities(facilities)
+      facilities.uniq do |facility|
+        [facility.address, facility.city, facility.state, facility.zip_code_5]
+      end
```

Comment:
would it be worth throwing on a sort here too just in case?
```suggestion
      end.sort_by { |facility| facility.distance }
```

Reply:
I haven't checked the API but figured maybe it would be a good check to have just in case. Fine to ship without! 👍

Reply:
I don't see the distance-based _ordering_ represented as a guarantee in the API documentation, so I would normally be inclined to hedge my bets by ordering it. That said, I don't think the current ordering is truly at risk of changing any time soon. Maybe we could get updated API docs that state the ordering as a guarantee.